contracts/amm/MinterAmm.sol

```diff
@@ -509,6 +509,8 @@ contract MinterAmm is InitializeableAmm, OwnableUpgradeSafe, Proxiable {
         bool sellTokens,
         uint256 collateralLeft
     ) internal returns (uint256) {
+        require(redeemer != address(0x0), "Invalid redeemer");
```

Comment:
Why do we need this check here? The method is private, so we can assume we'll always supply the address. Probably not a big deal, but adds a small amount of gas

Reply:
makes sense, removed